src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1136,9 +1121,7 @@ class CreateRoleStatementSegment(BaseSegment):
 
 
 @postgres_dialect.segment(replace=True)
-class ExplainStatementSegment(
-    ansi_dialect.get_segment("ExplainStatementSegment")  # type: ignore
-):
+class ExplainStatementSegment(ansi.ExplainStatementSegment):
```

Comment:
Here, we were already using real inheritance, but using a function to look up the base class. This caused `mypy` to complain, requiring a `# type: ignore` comment. Now we can remove that.